src/tool_operate.c

```diff
@@ -1442,6 +1442,46 @@ static CURLcode create_transfers(struct GlobalConfig *global,
             }
           }
 
+          /* In debug build of curl tool, using
+           *    --cert loadmem=<filename>:<password> --cert-type p12
+           *  must do the same thing than classic:
+           *    --cert <filename>:<password> --cert-type p12
+           *  but is designed to test curl_init_paramblob_struct_dup */
+#if defined(CURLDEBUG) || defined(DEBUGBUILD)
+          if(config->cert && (strlen(config->cert) > 8) &&
+             (memcmp(config->cert, "loadmem=",8) == 0)) {
+            FILE *fInCert = fopen(config->cert + 8, "rb");
+            void *certdata = NULL;
+            long filesize = 0;
+            int continue_reading = fInCert != NULL;
+            if(continue_reading)
+              filesize = fseek(fInCert, 0, SEEK_END) == 0;
```

Comment:
Shouldn't this be `continue_reading =` ?

Reply:
fixed